### DISCUSSION TITLE
When I try to update with an existing name in the database on the first click it shows me the warning message, but when I click again it does the update but saves me the blank record HELP!

### DISCUSSION THREAD

**Discussion by paginaartista** [prisma/prisma#18099] [Q&A]

**Comment by paginaartista** [comment]
When I try to update with an existing name in the database on the first click it shows me the warning message, but when I click again it does the update but saves me the blank record HELP!
![221026332-d1e5c209-1e51-4557-810b-f7efb75e0e48](https://user-images.githubusercontent.com/105748459/221367247-0072937f-08eb-40da-933d-5ce23da9bacf.png)

**Reply by nikolasburk** [reply]
Hey @paginaartista 👋 thanks a lot for raising this issue! It sounds like you might want to use a `@unique` attribute on the `fullName` field with this use case, did you consider this already?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
